src/components/icon.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
how about we call `/src/components` `/src/web-components` or `ui`

Reply:
I moved to `ui`